src/components/Inputs/InputBlock/InputBlock.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {
+	FunctionComponent,
+	useState,
+	useEffect,
+	ChangeEvent
+} from 'react';
+
+export interface ValidationMessageType {
+	message: string;
+	regex: RegExp;
+}
+
+export type InputBlockType = {
+	label?: string;
+	id?: string;
+	type: string;
+	className?: string;
```

Comment:
I would suggest change `className` to something else to prevent possible conflicts with React reversed prop.

Reply:
What is react reversed props? and I don't think there will be any conflict because this is a react prop and the className you're refering to is an attribute. Plus the react website uses this as well in examples. https://reactjs.org/docs/faq-styling.html

Reply:
My bad, typo.

Edited: I would suggest change className to something else to prevent possible conflicts with React reserved prop.

I might be wrong that it would make conflict. If it behaves fine during your development then the PR should be good to merge. Let me know.

They don't pass className from `prop` in the link btw.

Reply:
yeah, they don't. but i wanted to reduce confusion. and it works good in production.

Reply:
but let me make some changes first